packages/apollo-server/src/utils/exports.ts

```diff
@@ -0,0 +1,6 @@
+export * from 'graphql-tools';
+export * from 'graphql-subscriptions';
+export * from './types';
+// this makes it easy to get inline formatting and highlighting without
+// actually doing any work
+export const gql = String.raw;
```

Comment:
Does it make any sense to support `gql("blah")` (or `gql(string)`) to cover that use case?  (Not sure what `graphql-tag` does, or if we should allow that pattern.)